airflow/api_fastapi/execution_api/datamodels/taskinstance.py

```diff
@@ -122,15 +121,22 @@ class TIDeferredStatePayload(StrictBaseModel):
         ),
     ]
     classpath: str
-    trigger_kwargs: Annotated[dict[str, Any], Field(default_factory=dict)]
-    next_method: str
+    trigger_kwargs: Annotated[dict[str, Any] | str, Field(default_factory=dict)]
+    """
+    Kwargs to pass to the trigger constructor, either a plain dict or an encrypted string.
+
+    Both forms will be passed along to the trigger, the server will not handle either.
+    """
+
     trigger_timeout: timedelta | None = None
+    next_method: str
+    """The name of themethod on the operator to call in the worker after the trigger has fired."""
+    next_kwargs: Annotated[dict[str, Any] | str, Field(default_factory=dict)]
+    """
+    Kwargs to pass to the above method, either a plain dict or an ecnrypted string.
```

Comment:
```suggestion
    Kwargs to pass to the above method, either a plain dict or an encrypted string.
```

Reply:
Huh, I lost that change somehow. Thanks.